### PR TITLE
DEV-43932 Grafana upgrade code migration - Home dashboard

### PR DESCRIPTION
**Pull request by copyhold**
MIgrated:

* Home dashboard ability
* remove organizations links
* fixed the logzio services injection

### CODE REVIEW

File: public/app/features/org/OrgDetailsPage.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class OrgDetailsPage extends PureComponent<Props> {
   render() {
     const { navModel, organization } = this.props;
     const isLoading = Object.keys(organization).length === 0;
-    const canReadOrg = contextSrv.hasPermission(AccessControlAction.OrgsRead);
+    const canReadOrg = false; // LOGZ.IO GRAFANA CHANGE
```
Comment (yasmin-tr) [inline public/app/features/org/OrgDetailsPage.tsx]: No ticket reference? 😅 What functionality is it part of?
Btw with no original ticket I would put one for a new ticket perhaps or at least the ticket your merging this part of

File: packages/grafana-data/src/utils/logzioProvider.ts [hunk 2]
```diff
@@ -1,17 +1,8 @@
...
-if ((global as any).window) {
-  const logzioConfigurations = (window as any).logzio;
-  if (logzioConfigurations) {
-    logzioConfigs = logzioConfigurations.configs || {};
-    logzioDeprecatedAngularServices = logzioConfigurations.services || {};
-    productLoaded = logzioConfigurations.productLoaded || {};
-    if ((global as any).parent.__logzio__) {
-      logzioServices = (global as any).parent.__logzio__.services || {};
-    }
+export let logzioDeprecatedAngularServices: any = (global as any).parent.__logzio__.services;
```
Comment (ifatregev) [inline packages/grafana-data/src/utils/logzioProvider.ts]: usesd? it is the same as 'logzioServices'

File: packages/grafana-data/src/utils/logzioProvider.ts [hunk 3]
```diff
@@ -1,17 +1,8 @@
 // LOGZ.IO GRAFANA CHANGE
-export let logzioDeprecatedAngularServices: any = {};
-export let logzioServices: any = {};
-export let logzioConfigs: any = {};
-export let productLoaded: any = {};
```
Comment (ifatregev) [inline packages/grafana-data/src/utils/logzioProvider.ts:5]: don't need anymore?

**Review by ifatregev — APPROVED** [review]
Please check with mock data that nothing breaks and add info to description of PR so we will remember what was done if we need to upgrade again

File: packages/grafana-data/src/utils/logzioProvider.ts
Comment (copyhold) [inline packages/grafana-data/src/utils/logzioProvider.ts:1]: `productLoaded` call moved to the AppWrapper.tsx and is called directly from `window` object
`__logzio__.services` is already a Proxy and therefore always properly polulated.
logzio.configs is a json object that could be empty during iframe loading - wrap it with Proxy to always access up to date values

File: public/app/core/components/help/HelpModal.tsx
Comment (copyhold) [inline public/app/core/components/help/HelpModal.tsx:1]: remove shortcut

File: public/app/core/services/keybindingSrv.ts
Comment (copyhold) [inline public/app/core/services/keybindingSrv.ts:1]: remove shortcut

File: public/app/features/org/state/actions.ts
Comment (copyhold) [inline public/app/features/org/state/actions.ts:1]: hide organization from enduser

**Pull request closed without merging**